src/components/Slider/Slider.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Img from './style';
+import { PUBLIC_IMAGE_FOLDER, DEFAULT_BANNER_IMAGE, total } from '../../config/constants';
+import { getRandomNumber, getNextRoundRobin } from '../../libs/utils/math';
+
+class Slider extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      current: -1,
+
+    };
+  }
+
+  componentDidMount = () => {
+    const { random, duration } = this.props;
+    let { current } = this.state;
+    this.id = setInterval(() => {
+      if (random) {
+        current = getRandomNumber(total);
+      } else {
+        current = getNextRoundRobin(current, total);
+      }
+      this.setState({ current });
+    }, duration);
+  }
+
+  componentWillUnmount =() => {
+    clearInterval(this.id);
+  }
+
+  render() {
+    const { current } = this.state;
+    const {
+      altText, height, duration, banner,
+    } = this.props;
+    const { defaultbanner } = this.props;
+    if (current === -1 || banner.length === 0) {
+      return (
+        <>
+          <div>
+            <Img src={`${PUBLIC_IMAGE_FOLDER}${defaultbanner}`} alt={altText} height={height} duration={duration} />
```

Comment:
no need to add ${PUBLIC_IMAGE_FOLDER} as we are getting the default banner from props